schematic/store/synapse.py

```diff
@@ -2825,7 +2825,7 @@ def getDatasetAnnotations(
             try:
                 logger.info("Trying batch mode for retrieving Synapse annotations")
                 table = self.getDatasetAnnotationsBatch(datasetId, dataset_file_ids)
-            except (SynapseAuthenticationError, SynapseHTTPError):
+            except:
```

Comment:
This is not great practice - do you know what other exceptions you ran into here?

Reply:
The one in this case was a `ValueError`

Reply:
@GiaJordan and everyone else, we can add this multiblock exception: https://rollbar.com/blog/python-catching-multiple-exceptions/.

if it makes sense to add `ValueError` we should

Reply:
I should also add that thanks for being proactive about fixing that.  I agree that we can fix things on the fly so every commit to the codebase is better!

Reply:
So `valueError` should no longer be raised, that's what this fix is for and the bug the user experienced. However, the previous behavior that the user experienced was that the program would stop since `ValueError` was not one of `SynapseAuthenticationError` or `SynapseHTTPError`. 
Since none of the other methods used later in the `try` block raise exceptions themselves, my thinking was that if there were any issue with using batch mode, then sequential mode should be used and if another exception was raised during that process it would be surfaced to the user like normal, but that it would allow the user to actually generate a manifest in cases where there isn't an issue with the sequential method instead of repeatedly getting errors and being unable to proceed. At this point I don't believe we need to also catch `ValueError` given this fix, I was just worried that the two listed were too restricted given the behavior the user experienced in this case.
@thomasyu888

Reply:
Thanks @GiaJordan , I'm not opposed to add more exceptions, but accepting all exceptions is a bit scary (imo)

so something like
```
except (SynapseAuthenticationError, SynapseHTTPError, ValueError)
```

Reply:
> accepting all exceptions is a bit scary (imo)

Yeah I can see that and I agree. I was weighing the behavior of not blocking generation vs being specific because at this point I'm not sure which other exceptions could be raised depending on what users do (like how in this case this user flow wasn't expected.

In the end though we can add `ValueError` to be safe, though I don't think it's terribly necessary with this change, and we can add to the list of exceptions (and possibly add other blocks) as necessary in the future.

Thanks for this discussion! @thomasyu888